chain/beacon/node.go

```diff
@@ -234,6 +234,9 @@ func (h *Handler) Transition(prevGroup *key.Group) error {
 
 // TransitionNewGroup prepares the node to transition to the new group
 func (h *Handler) TransitionNewGroup(newShare *key.Share, newGroup *key.Group) {
+	if h == nil {
```

Comment:
Without this change, there are cases when a test will fail because the handler is nil. Maybe it's worth investigating the issue more in-depth rather than fix the error?

Reply:
Tracked in https://github.com/drand/drand/issues/1112